content/blogposts/2023/release-5.1.md

```diff
@@ -0,0 +1,120 @@
+
+#### Precise cross-repository code navigation for C and C++
+
+Sourcegraph’s latest SCIP indexer, [scip-clang](https://sourcegraph.com/github.com/sourcegraph/scip-clang), adds precise code navigation for C and C++. C and C++ precise code navigation works for all features supported by scip-clang, such as types, methods, macros, forward declarations, and more. Cross-repository navigation is also supported, so if you reference a function outside of your current repository, you can jump directly to its definition as long as it’s indexed.
+
+Site admins can [add precise code navigation](https://docs.sourcegraph.com/code_navigation/how-to/adding_lsif_to_many_repos) to their Sourcegraph instance by running the SCIP indexers as a periodic job in CI infrastructure. 
+
+<br/>
+  
+<Video 
+  source={{
+    mp4: 'blog/release-post/5.1/codenav_gif',
+  }}
+  loop={true}
+  title="scip-clang"
+/>  
+  
+<br />
+<br/><br/>
+
+<Badge text="Code graph" color="green" size="small" />
+
+#### Ownership inference
+
+Sourcegraph [started ingesting code ownership data](https://about.sourcegraph.com/blog/our-vision-for-code-ownership), including CODEOWNERS, in 5.0, and moves from experimental to beta in this release. It’s not always easy to assign ownership to code, and sometimes the data is out of date—in 5.1, Sourcegraph also infers code ownership from code and code host metadata (e.g. recent commits). Read more about how we help maintain ownership coverage from our [tech lead’s blog post](https://about.sourcegraph.com/blog/boosting-code-ownership). 
+
+Site admins can [turn on ownership signals](https://docs.sourcegraph.com/own#enabling-sourcegraph-own) by going to ```Site Admin > Code Graph > Ownership Signals``` and enabling recent-contributors and recent-views. 
+
+<br/>
+  
+<Video 
+  source={{
+    mp4: 'blog/release-post/5.1/own_gif3',
+  }}
+  loop={true}
+  title="Ownership inference"
+/>  
+  
+<br />
+<br/><br/>
+
+<Badge text="Code graph" color="green" size="small" />
+
+#### Custom repository metadata 
+
+Custom repository metadata brings user-defined key-value pairs into the code graph, so users can filter search results based on custom metadata they define. For example, metadata could indicate a repository's maintenance status, allowing users to exclude deprecated or unmaintained libraries from their results. 
+
+Repository metadata can be [added through the web UI, src-cli and GraphQL API](https://docs.sourcegraph.com/admin/repo/metadata). 
+
+<br/>
+  
+<Video 
+  source={{
+    mp4: 'blog/release-post/5.1/metadata_gif2',
+  }}
+  loop={true}
+  title="Custom repository metadata"
+/>  
+  
+<br />
+<br/><br/>
+
+<Badge text="Code graph" color="green" size="small" />
+
+#### Complete Perforce support
+
+We see many teams using Perforce in combination with Git, and until recently, they lacked a single experience. Now, devs can search and navigate code hosted in either version control systems in Sourcegraph.
```

Comment:
"lacked a single experience" is not clear. Also, "either version control systems" - should not be plural, right? Or should be "both" not "either"?